source/core/src/main/com/csse3200/game/components/player/PlayerAnimationController.java

```diff
@@ -16,9 +16,9 @@ public class PlayerAnimationController extends Component {
     private Direction previousDirectionHorizontal;
 
     /**
-     Indicate the selected player.
+     * Indicate the selected player.
      */
-    private enum PlayerNum {
+    public enum PlayerNum {
```

Comment:
This file belongs to team 5, so you should not change anything as it may affects their functionality. PlayerStatsDisplay is the only exception.